actionpack/test/dispatch/ssl_test.rb

```diff
@@ -98,8 +98,8 @@ def assert_post_redirected(redirect: {}, from: "http://a/b?c=d",
 end
 
 class StrictTransportSecurityTest < SSLTest
-  EXPECTED = "max-age=15552000"
-  EXPECTED_WITH_SUBDOMAINS = "max-age=15552000; includeSubDomains"
+  EXPECTED = "max-age=31536000"
+  EXPECTED_WITH_SUBDOMAINS = "max-age=31536000; includeSubDomains"
```

Comment:
Same here:

`"max-age=#{1.year.to_i}"` as in https://github.com/rails/rails/blob/3b4f14dc43f09addb50c546ffe23b9e03eafed20/railties/lib/rails/generators/rails/app/templates/config/environments/development.rb.tt#L22

Reply:
@6temes Not sure if the intention behind https://github.com/rails/rails/commit/5d149048ffa56c8fa1c41d1ef3bcf9472052f369 applies in the tests as well.